codes/ruby/chapter_searching/binary_search_edge.rb

```diff
@@ -0,0 +1,65 @@
+=begin
+File: binary_search_edge.rb
+Created Time: 2024-04-09
+Author: Blue Bean (lonnnnnnner@gmail.com)
+=end
+
+### 二分查找插入点（存在重复元素） ###
+def binary_search_insertion(nums, target)
```

Comment:
If I use require_relative, then the driver Code in binary_search_insertion.rb may be like:

```
### Driver Code ###
if __FILE__ == $0
  # Code
end
```

Otherwise the driver code in `binary_search_insertion.rb` will be executed when executing `binary_serch_edge.rb`.

So may I make this change?

Reply:
Yes, it is. Actually, at the very first I did not expect the case that we will import another method from an index file. I will create another PR to put all the existing driver code at branch main into the `__FILE__ == 0` block.